### PR TITLE
Added recipe for DoctrinePHPCRBundle

### PR DESCRIPTION
**Pull request by giovannialbero1992**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: doctrine/phpcr-bundle/1.3/manifest.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "bundles": {
+        "Doctrine\\Bundle\\PHPCRBundle\\DoctrinePHPCRBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "src/": "%SRC_DIR%/"
```
Comment (Nyholm) [inline doctrine/phpcr-bundle/1.3/manifest.json]: Can you explain why we need to create a `Document` folder?

Comment (giovannialbero1992) [reply doctrine/phpcr-bundle/1.3/manifest.json]: I've created `Document` folder to be compliant with doc http://symfony.com/doc/master/cmf/bundles/phpcr_odm/models.html

Comment (giovannialbero1992) [reply doctrine/phpcr-bundle/1.3/manifest.json]: If you consider is not necessary for this recipe i'll remove it.

Comment (Nyholm) [reply doctrine/phpcr-bundle/1.3/manifest.json]: I'm not familiar with the CMF, that is why Im asking questions =)

Am I required to put the documents in src/Document or could I as easily put them in src/Foo/Bar/Baz?

Comment (giovannialbero1992) [reply doctrine/phpcr-bundle/1.3/manifest.json]: You have right. I remove this part.

**Comment by Nyholm** [comment]
Im still having the same issue

**Comment by giovannialbero1992** [comment]
@Nyholm we'll wait next release. I've tried this recipe with dev-master (play it safe 😅 ), when a further release will be released i will do an update on this PR.

**Comment by Nyholm** [comment]
Hey @giovannialbero1992, any updates?

**Comment by giovannialbero1992** [comment]
@Nyholm the new version has still not been released

**Comment by Nyholm** [comment]
Any updates now?

**Comment by giovannialbero1992** [comment]
@Nyholm No for the moment. Now there is a pre-release.

**Comment by Nyholm** [comment]
Im still waiting with excitement =)

**Comment by giovannialbero1992** [comment]
@Nyholm I'm hitting my head against the wall 😅... I've reached a PR that has one test green and one red.

**Pull request merged**